package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ng-sidebar",
-  "version": "9.2.0",
+  "version": "9.2.1",
```

Comment:
Can you remove this from this PR? I'll bump it up separately as part of the release process.

Reply:
Sure, no problem.